src/Apps/Order/Routes/Shipping2/FulfillmentDetails.tsx

```diff
@@ -39,11 +39,11 @@ import { useFeatureFlag } from "System/useFeatureFlag"
 import { SavedAddressesFragmentContainer as SavedAddresses } from "Apps/Order/Routes/Shipping2/SavedAddresses2"
 import { usePrevious } from "Utils/Hooks/usePrevious"
 import { Collapse } from "Apps/Order/Components/Collapse"
-import { useShippingContext } from "Apps/Order/Routes/Shipping2/ShippingContext"
+import { useShippingContext } from "Apps/Order/Routes/Shipping2/support/ShippingContext"
```

Comment:
Never seen a "support" directory concept in force before, so lets not introduce new patterns

Reply:
ah, i think i just brought that over from rails apps. i'm not stuck on the name, just wanted to start bringing some order to this folder, maybe not necessary right now as you said

Reply:
Not necessary, but if it were, `utils` go in `Utils`, Context lives in root, `Components` live in components, hooks go in `Hooks`. We don't need any other folders (though `Queries` and `Mutations` could be considered an exception). If there's ever any question, the `Apps/Auction` app is canonical.